### ISSUE TITLE
Emulator is broken

### ISSUE THREAD

**Issue by codeanticode** [processing/processing-android#564]
After downloading the system image, it give the following error in the console:

```
Emulator process exited with status 1.
EmulatorController: Emulator never booted. NOT_RUNNING
Error while starting the emulator. (NOT_RUNNING)
```

**Comment by codeanticode** [comment]
The problem seems to be the the new location of the emulator tool, as described [here](https://www.stkent.com/2017/08/10/update-your-path-for-the-new-android-emulator-location.html).

**Comment by codeanticode** [comment]
Fixed with https://github.com/processing/processing-android/commit/d6908c8b8a1479e7d9b56b0a9ffeca5c21732e68

**Comment by whatmood** [comment]
> Fixed with [d6908c8](https://github.com/processing/processing-android/commit/d6908c8b8a1479e7d9b56b0a9ffeca5c21732e68)
I have meet the same issue. Could you tell how to fix it? any step to follow about what you said "Fixed with d908c8". thanks!
